firefox-ios/firefox-ios-tests/Tests/XCUITests/AuthenticationTest.swift

```diff
@@ -46,12 +46,8 @@ class AuthenticationTest: BaseTestCase {
          inaccessible after sign in to verify the success text. */
         waitForNoExistence(app.alerts.buttons["Cancel"], timeoutValue: 5)
         waitForNoExistence(app.alerts.buttons["Log in"], timeoutValue: 5)
-        // Added this check to ensure the BasicAuth login is persisting after app restart as well.
-        app.terminate()
-        app.launch()
         navigator.openURL(testBasicHTTPAuthURL)
```

Comment:
For some reason, I have to reload the URL after entering the username and password to have the website loaded. I can reproduce the same issue from one other basic auth testing site (https://httpbin.org/#/Auth/get_basic_auth__user___passwd_).

Bug or not a bug?

Reply:
If you ask me, I would say is a bug but maybe we should ask the product team

Reply:
I assume this is the bug reported by you @clarmso for this: https://github.com/mozilla-mobile/firefox-ios/issues/23565

Reply:
i am in favor for an workaround and make this test pass, but lets make sure after the bug is fixed the test fails in order to remove it

Reply:
I would say it is a bug since this test has been passing consistently